Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Aquality.Selenium.Template.NUnit.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Let's migrate to .NET 6, .NET Core 3.1 has End of Support date in the end of the current year 
https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core